fuels-abigen-macro/tests/harness.rs

```diff
@@ -1274,12 +1274,16 @@ async fn test_gas_errors() {
     // Test for insufficient gas.
     let result = contract_instance
         .initialize_counter(42) // Build the ABI call
-        .tx_params(TxParameters::new(Some(1_000), Some(100), None))
+        .tx_params(TxParameters::new(
+            Some(DEFAULT_INITIAL_BALANCE),
+            Some(100),
+            None,
+        ))
         .call() // Perform the network call
         .await
         .expect_err("should error");
 
-    assert_eq!("Contract call error: Response errors; unexpected block execution error InsufficientGas { provided: 100, required: 100000 }", result.to_string());
+    assert_eq!("Contract call error: Response errors; unexpected block execution error InsufficientGas { provided: 1000000000, required: 100000000000 }", result.to_string());
```

Comment:
Why is this error about insufficient gas and not balance???

Reply:
IIRC, because that's what the node returns given the configuration parameters (`DEFAULT_INITIAL_BALANCE x 100`). It's just testing `tx_params` is properly setting things in the transaction.